frontend/tests/unit/components/Version.spec.ts

```diff
@@ -2,29 +2,57 @@ import { mount } from '@vue/test-utils';
 
 import Version from '@/components/Version.vue';
 
-const envCopy = Object.assign({}, APP_RELEASE);
+const globalCopy = Object.assign({}, APP_RELEASE);
+const importEnvCopy = Object.assign({}, import.meta.env);
 
-const resetEnvMock = () => {
-  Object.assign(APP_RELEASE, envCopy);
+const resetImportEnv = (): void => {
+  Object.assign(import.meta.env, importEnvCopy);
+};
+
+const resetGlobalVariables = (): void => {
+  Object.assign(APP_RELEASE, globalCopy);
 };
 
 describe('Version.vue', () => {
   afterEach(() => {
-    resetEnvMock();
+    resetImportEnv();
+    resetGlobalVariables();
+  });
+
+  it('is hidden when in development mode', () => {
+    Object.assign(import.meta.env, {
+      MODE: 'development',
+    });
+    const wrapper = mount(Version);
+
+    const commitHashEl = wrapper.find('[data-test="version"]');
+    expect(commitHashEl.exists()).toBe(false);
   });
 
-  it('is hidden when no version is defined', () => {
+  it('is visible when in production mode', () => {
+    Object.assign(import.meta.env, {
+      MODE: 'production',
+    });
```

Comment:
Does this mean the commit hash/version won't be visible in development mode? If so, why would we hide it?

Reply:
Because it introduces a few problems and doesn't provide any value for the developers.
1. When working locally on the frontend and you are several commits ahead of upstream branch, the link will not work as that commit hash cannot be found upstream. This will not happen on staging/production deployments as we are usually deploying from our github repo.
2. I don't see any benefit of having it in dev mode. What this version string refers to is the `Build version` of the deployed app.